### DISCUSSION TITLE
`brew audit --online --new-formula` fails for `uses_from_macos [...], since: :monterey`

### DISCUSSION THREAD

**Discussion by pavelzw** [Homebrew/discussions#4130] [Everyday usage]
I am currently [trying](https://github.com/Homebrew/homebrew-core/pull/120924) to build a formula for micromamba. There, I have the issue that micromamba uses needs `curl >= 7.66.0` while macOS 11 ships an older version. To fix this, I added `uses_from_macos "curl", since: monterey` (same thing with `libarchive`). Now, `brew audit --online --new-formula micromamba` returns the following error:
```
==> brew audit micromamba --online --new-formula
==> FAILED
Full audit micromamba --online --new-formula output
  micromamba:
    * Dependency 'curl' is provided by macOS; please replace 'depends_on' with 'uses_from_macos'.
    * Dependency 'libarchive' is provided by macOS; please replace 'depends_on' with 'uses_from_macos'.
  Error: 2 problems in 1 formula detected
```

This error only occurs in the [macOS 11 CI runs](https://github.com/Homebrew/homebrew-core/actions/runs/3990623199/jobs/6844571586#step:6:76).

How can I make brew stop complaining?

**Comment by oregonpillow** [comment]
You could try adding it to exception list, like here #4623
